lang/po/new/cs.po

```diff
@@ -383,12 +380,12 @@ msgstr "Levy vent.:"
 # MSG_INFO_EXTRUDER c=18
 #: ultralcd.cpp:2040
 msgid "Extruder info"
-msgstr ""
+msgstr "Extruder info"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.